### ISSUE TITLE
Change search UI 

### ISSUE THREAD

**Issue by Seandylan123** [NeoApplications/Neo-Launcher#302]
The search UI looks more of a android 11 style
![IMG_20221216_173156](https://user-images.githubusercontent.com/106500241/208068810-b7f95041-f926-4678-be40-831800bbead6.jpg)
This is what it looks like
![images (2)](https://user-images.githubusercontent.com/106500241/208068882-2a8196ed-4408-46e9-ac1e-444be77e5e5c.jpeg)
And it should probably have to look like this

**Comment by machiav3lli** [comment]
Neo Launcher is still based on AOSP 12 and we'll start migration to AOSP 13 next year, which should bring some of those MD3 changes with it.

**Comment by Seandylan123** [comment]
> Neo Launcher is still based on AOSP 12 and we'll start migration to AOSP 13 next year, which should bring some of those MD3 changes with it.

It does not make any sense since AOSP 12 already has MD3 on their search bar

**Comment by Seandylan123** [comment]
> > Neo Launcher is still based on AOSP 12 and we'll start migration to AOSP 13 next year, which should bring some of those MD3 changes with it.
> 
> It does not make any sense since AOSP 12 already has MD3 on their search bar

Also neo launcher settings has MD3

**Comment by machiav3lli** [comment]
@Seandylan123 nope, it doesn't. Just check other AOSP based launchers (other than the pixel launcher, maybe a simple ROM launcher would do it). Plus, we wrote the whole settings ourselves...
